cmd/broker/fanout/main.go

```diff
@@ -68,6 +72,8 @@ func main() {
 		logger.Fatalf("MAX_STALE_DURATION must be greater than pool resync period %v", poolResyncPeriod)
 	}
 
+	profiling.StartGCPProfiler("broker-fanout", env.GCPProfilerEnvConfig)
```

Comment:
Can we move this to the mainhelper.Init as an option (line 67 above)?

Reply:
Thanks for pointing this out! I see we already use knative/pkg's pprof handler. I'll move gcp profiler init here.

Reply:
Ok, this should now be configured automatically for any command that uses mainhelper.Init. Since the available env vars are hidden in the profiling package, I added docs in `docs/development/profiling.md` to explain what env vars may be used.